### DISCUSSION TITLE
Ability to swap sidebar item order

### DISCUSSION THREAD

**Discussion by mindaugaswashere** [runelite/runelite#18302] [Ideas]
![image](https://github.com/user-attachments/assets/3200a8b4-793c-488a-9a33-1eaea8f2c144)
I find it bit confusing when I add some plugins and they appear in random place messing up prioritized ones, would be nice to be able reorder them with drag&drop ability.

**Comment by cdfisher** [comment]
I believe this is a duplicate of #15020 

Accidentally deleted my comment going to edit it.
